src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -6059,6 +6059,56 @@ static const int64_t right_3_bits = right_n_bits(3);
     return start;
   }
 
+  void generate_vector_math_stubs() {
+    if (UseRVV) {
```

Comment:
Seems to me cleaner to do this:
```
if (UseRVV) {
  generate_vector_math_stubs();
}
```

Reply:
As there are several log output in the method, I think it might be better to put them together in this method.

But I made some modification so the indentation looks better.